### PR TITLE
added more options to resolve.alias

### PR DESCRIPTION
**Pull request by niravasher**
This PR was open for some time, with unresolved changes #3358 , closes this one

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/

### CODE REVIEW

**Comment by niravasher** [comment]
@jamesgeorge007 @montogeek @EugeneHlushko what else is required to merge the PR? Please suggest required changes

File: src/content/configuration/resolve.md [hunk 1]
```diff
@@ -39,7 +40,7 @@ module.exports = {
 
 ### `resolve.alias`
 
-`object`
+`object` `[string]` `boolean: false`
```
Comment (anikethsaha) [inline src/content/configuration/resolve.md]: It doesn't take a `string` as a value I guess. Please check its `key : value` pair

Comment (anikethsaha) [reply src/content/configuration/resolve.md]: I think you meant
`resolve.alias[key] : [string] / false` ? if so, please change the docs accordingly

**Comment by niravasher** [comment]
@anikethsaha Changes have been done by @jeffin143

**Pull request closed without merging**